### ISSUE TITLE
Fails to synchronize configs other than installed extension

### ISSUE THREAD

**Issue by kwonoj** [shanalikhan/code-settings-sync#98]
**Visual Studio Code Version :** 1.5.2
**Code Sync Settings Version :** 2.2.3
**Operating System :** Mac OS 10.11.6

Please make sure to write the versions and console logged error picture or copied text.

Console log shows below error message.

```
Cannot read property 'content' of undefined: TypeError: Cannot read property 'content' of undefined
    at /Users/ojkwon/.vscode/extensions/Shan.code-settings-sync-2.2.3/out/src/extension.js:362:101
    at next (native)
    at /Users/ojkwon/.vscode/extensions/Shan.code-settings-sync-2.2.3/out/src/extension.js:7:65
    at __awaiter (/Users/ojkwon/.vscode/extensions/Shan.code-settings-sync-2.2.3/out/src/extension.js:3:12)
    at /Users/ojkwon/.vscode/extensions/Shan.code-settings-sync-2.2.3/out/src/extension.js:313:32
    at process._tickCallback (internal/process/next_tick.js:103:7)e.onUnexpectedError @ shell.ts:428
```

**Comment by shanalikhan** [comment]
would you please post the images of the settings folder the files and snippets folders.

Also check 
is there any settings files values are null , not even single space is saved ?

**Comment by shanalikhan** [comment]
I have fixed this in 2.2.5. Would you check it out :)

**Comment by kwonoj** [comment]
Thanks, will try.
